### ISSUE TITLE
Add {Servertime} and {date}

### ISSUE THREAD

**Issue by WGSRangerWolf** [uEssentials/uEssentials#465]
Make it possible to use this message in auto announcer,

The server time is {servertime} and {serverdate}

**Comment by Aceheliflyer** [comment]
And how long it's been running?

**Comment by WGSRangerWolf** [comment]
Like {uptime} @Aceheliflyer?  and {online} to view how many is on

**Comment by Aceheliflyer** [comment]
Yes, and I also think it would be helpful to add the server uptime to `/stime` too.

**Comment by leonardosnt** [comment]
Can you give me an example?

**Comment by Aceheliflyer** [comment]
[Alert] Server: 11/3/2016 6:52:13 PM (Running since 11/2/2016 5:34:25 PM)

**Comment by WGSRangerWolf** [comment]
[Broadcast] Server: 11/3/2016 6:00:10 PM (Running since: 11/3/2016 5:00:00 PM, running for: 0d 1h 0m 10s)

**Comment by OldCash4u** [comment]
PT-BR

Apoio ! :+1:
